Readme.md

```diff
@@ -7,6 +7,9 @@ A Ruby gem version of [https://github.com/tchapi/pagedown-bootstrap](https://git
 Add to your `Gemfile`:
 
     gem 'pagedown-bootstrap-rails'
+    gem 'bootstrap-sass' # check version
+    gem 'sass-rails', # check version
```

Comment:
Do you mind removing this comma? The comments about checking the versions are probably superfluous too. Thx.